### PR TITLE
Finalize 4.0.0 (version code update + CHANGELOG)

### PR DESCRIPTION
**Pull request by felixdivo**

### CODE REVIEW

**Comment by felixdivo** [comment]
@hardbyte Is it correct that we only release from master? If yes, it's time for a (big?) merge. Do you have time for that?

File: can/__init__.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 import logging
 from typing import Dict, Any
 
-__version__ = "4.0.0-rc.0"
+__version__ = "4.0.0"
```
Comment (hardbyte) [inline can/__init__.py:11]: 👀

**Review by hardbyte — APPROVED** [review]

**Comment by hardbyte** [comment]
Sure - I'll merge this from develop into master.

**Comment by hardbyte** [comment]
Done

**Comment by felixdivo** [comment]
Thank you! 🥳

**Pull request merged**